public/app/plugins/panel/timeseries/migrations.ts

```diff
@@ -352,6 +376,49 @@ export function graphToTimeseriesOptions(angular: any): { fieldConfig: FieldConf
     }
   }
 
+  // timeRegions migration
+  if (angular.timeRegions?.length) {
+    let regions: any[] = angular.timeRegions.map((old: GraphTimeRegionConfig, idx: number) => ({
+      name: `T${idx + 1}`,
+      color: old.colorMode !== 'custom' ? old.colorMode : old.fillColor,
+      line: old.line,
+      fill: old.fill,
+      fromDayOfWeek: old.fromDayOfWeek,
+      toDayOfWeek: old.toDayOfWeek,
+      from: old.from,
+      to: old.to,
+    }));
+
+    regions.forEach((region) => {
+      const queryTarget: GrafanaQuery = {
+        queryType: GrafanaQueryType.TimeRegions,
+        refId: 'Anno',
+        timeRegion: {
+          fromDayOfWeek: region.fromDayOfWeek,
+          toDayOfWeek: region.toDayOfWeek,
+          from: region.from,
+          to: region.to,
+          timezone: getDashboardSrv().getCurrent()?.getTimezone(),
+        },
+      };
+
+      annotations.push({
+        datasource: {
+          type: 'datasource',
+          uid: 'grafana',
+        },
+        enable: true,
```

Comment:
```suggestion
        enable: true,
        hidden: true,  // don't show the toggle at the top of the dashboard
```

Reply:
hm, isn't that a bit restrictive? in old graph you can set the fill to false to be able to hide them 🤔